src/queries/page.server.ts

```diff
@@ -35,6 +38,145 @@ export const fetchGetPage = async (
   })
 }
 
+export const getImageDimensionByUri = async (uri: string) => {
+  return new Promise<{ width: number; height: number; uri: string }>(
+    (resolve, reject) => {
+      const options = url.parse(uri)
+
+      https.get(options, function (response) {
```

Comment:
It is best to use fetch here. On one hand, it is a more modern API. On the other hand, Next.js has some optimizations for fetch.

Reply:
Got it! Thanks!